cmd/security-agent/subcommands/start/command_test.go

```diff
@@ -27,7 +27,6 @@ func TestCommand(t *testing.T) {
 			check: func(cliParams *cliParams, params core.BundleParams) {
 				// Verify logger defaults
 				require.Equal(t, command.LoggerName, params.LoggerName(), "logger name not matching")
-				require.Equal(t, "info", params.LogLevelFn(nil), "log level not matching")
```

Comment:
These are seg-faulting because the func signature for LogForDaemon `func LogForDaemon(loggerName, logFileConfig, defaultLogFile string) Params` does not have the `overrideFromEnv` input param from LogForOneShot `func LogForOneShot(loggerName, level string, overrideFromEnv bool) Params`. 

`If overrideFromEnv is set, then DD_LOG_LEVEL will override the given level.`

I believe LogForDaemon should also have this param. Asking shared components about this.

Reply:
Ok after chatting with the components team, we can remove this line because we no longer need to ensure that a correct default value is set.

```
LogForOneShot is used for agent commands which sometime need to disable the logger. For example [hostname](https://github.com/DataDog/datadog-agent/blob/49a0dc97687513cbdfe1bcd70d5b57b827d86f2a/cmd/agent/subcommands/hostname/command.go#L43) command.
LogForDaemon was designed to be used when running daemon / service which should rely on the user configuration.
```

By design we do NOT want the DD_LOG_LEVEL env var to be able to override the config entry when using LogForDaemon, so the seg-faulting is expected